src/content/developers/docs/bridges/index.md

```diff
@@ -0,0 +1,139 @@
+
+## Benefits of bridges {#benefits-of-bridges}
+
+Put simply, bridges unlock numerous use cases by allowing blockchain networks to exchange data and move assets between them.
+
+Blockchains have unique strengths, weaknesses, and approaches to building applications (such as speed, throughput, costliness, etc.). Bridges help the development of the overall crypto ecosystem by enabling blockchains to leverage the innovations of each other.
+
+For developers, bridges enable the following:
+
+- the transfer of any data, information, and assets across chains.
+- unlocking new features and use cases for protocols as bridges expand the design space for what protocols can offer. For example, a protocol for yield farming originally deployed on Ethereum Mainnet can offer liquidity pools across all EVM-compatible chains.
+- the opportunity to leverage the strengths of different blockchains. For example, developers can benefit from the lower fees offered by the different L2 solutions by deploying their dapps across rollups, and sidechains and users can bridge across them.
+- collaboration among developers from various blockchain ecosystems to build new products.
+- attracting users and communities from various ecosystems to their dapps.
+
+## How do bridges work? {#how-do-bridges-work}
+
+While there are many [types of bridge designs](https://blog.li.fi/what-are-blockchain-bridges-and-how-can-we-classify-them-560dc6ec05fa), three ways to facilitate the cross-chain transfer of assets stand out:
+
+- **Lock and Mint –** Lock assets on the source chain and mint assets on the destination chain.
+- **Burn and Mint –** Burn assets on the source chain and mint assets on the destination chain.
+- **Atomic Swaps –** Swap assets on the source chain for assets on the destination chain with another party.
+
+## Bridge types {#bridge-types}
+
+Bridges can usually be classified into one of the following buckets:
+
+- **Native Bridges –** These bridges are typically built to bootstrap liquidity on a particular blockchain, making it easier for users to move funds to the ecosystem. For example, the [Arbitrum Bridge](https://bridge.arbitrum.io/) is built to make it convenient for users to bridge from Ethereum Mainnet to Arbitrum. Other such bridges include Polygon PoS Bridge, [Optimism Gateway](https://gateway.optimism.io/), etc.
+- **Validator or Oracle Based Bridges –** These bridges rely on an external validator set or oracles to validate cross-chain transfers. Examples: Multichain and Across.
+- **Generalized Message Passing Bridges –** These bridges can transfer assets, along with messages and arbitrary data across chains. Examples: Nomad and LayerZero.
+- **Liquidity Networks –** These bridges primarily focus on transferring assets from one chain to another via atomic swaps. Generally, they don’t support cross-chain message passing. Examples: Connext and Hop.
```

Comment:
Just noticing we switch between title and sentence casing on these bolded bullet points. 
Could be nice to tidy these up to all use sentence casing per the style guide
```suggestion
- **Native bridges –** These bridges are typically built to bootstrap liquidity on a particular blockchain, making it easier for users to move funds to the ecosystem. For example, the [Arbitrum Bridge](https://bridge.arbitrum.io/) is built to make it convenient for users to bridge from Ethereum Mainnet to Arbitrum. Other such bridges include Polygon PoS Bridge, [Optimism Gateway](https://gateway.optimism.io/), etc.
- **Validator or oracle based bridges –** These bridges rely on an external validator set or oracles to validate cross-chain transfers. Examples: Multichain and Across.
- **Generalized message passing bridges –** These bridges can transfer assets, along with messages and arbitrary data across chains. Examples: Nomad and LayerZero.
- **Liquidity networks –** These bridges primarily focus on transferring assets from one chain to another via atomic swaps. Generally, they don’t support cross-chain message passing. Examples: Connext and Hop.
```